src/Symfony/Component/HttpKernel/Tests/EventListener/TestSessionListenerTest.php

```diff
@@ -123,7 +123,7 @@ public function testSessionWithNewSessionIdAndNewCookieDoesNotSendAnotherCookie(
 
         $response = $this->filterResponse(new Request(), HttpKernelInterface::MASTER_REQUEST, $response);
 
-        $this->assertSame($expected, $response->headers->get('Set-Cookie', null, false));
+        $this->assertSame($expected, $response->headers->all()['set-cookie']);
```

Comment:
why not `all('set-cookie')` now we can?

Reply:
It does not work as expected in low/high deps because it does not get this version of the PR, maybe there is something better to do ?

Reply:
i think it will work after merging upstream

https://github.com/symfony/symfony/blob/40fe1610d23ae84f99561c45c118f58bba9e19c9/src/Symfony/Component/HttpKernel/composer.json#L23

Reply:
yes but the tests fails before in this PR because of this :/, maybe we can reupdate the tests when everything is in upstream

Reply:
:+1: no real blocker anyway